front/pages/api/w/[wId]/labs/transcripts/default.ts

```diff
@@ -0,0 +1,83 @@
+import type { WithAPIErrorReponse } from "@dust-tt/types";
+import { assertNever } from "@dust-tt/types";
+import type { NextApiRequest, NextApiResponse } from "next";
+
+import { Authenticator, getSession } from "@app/lib/auth";
+import { LabsTranscriptsConfigurationResource } from "@app/lib/resources/labs_transcripts_resource";
+import { apiError, withLogging } from "@app/logger/withlogging";
+import type { GetLabsTranscriptsConfigurationResponseBody } from "@app/pages/api/w/[wId]/labs/transcripts";
+
+async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<
+    WithAPIErrorReponse<GetLabsTranscriptsConfigurationResponseBody>
+  >
+): Promise<void> {
+  const session = await getSession(req, res);
+  const auth = await Authenticator.fromSession(
+    session,
+    req.query.wId as string
+  );
+
+  const owner = auth.workspace();
+  const userId = auth.user()?.id;
+
+  if (!owner || !userId) {
+    return apiError(req, res, {
+      status_code: 404,
+      api_error: {
+        type: "workspace_not_found",
+        message: "The workspace or user was not found.",
+      },
+    });
+  }
+
+  if (!owner.flags.includes("labs_transcripts")) {
+    return apiError(req, res, {
+      status_code: 403,
+      api_error: {
+        type: "feature_flag_not_found",
+        message: "The feature is not enabled for this workspace.",
+      },
+    });
+  }
+
+  switch (req.method) {
+    case "GET":
+      const transcriptsConfigurationGet =
+        await LabsTranscriptsConfigurationResource.findByWorkspace({
+          auth,
+        });
+
+      if (!transcriptsConfigurationGet) {
+        return apiError(req, res, {
+          status_code: 404,
+          api_error: {
+            type: "agent_configuration_not_found",
+            message: "The configuration was not found.",
+          },
+        });
+      }
+
+      switch (transcriptsConfigurationGet.provider) {
+        case "google_drive":
+          return apiError(req, res, {
+            status_code: 404,
+            api_error: {
+              type: "transcripts_configuration_default_not_allowed",
+              message: "The provider does not allow default configurations.",
+            },
+          });
+        case "gong":
+          break;
+        default:
+          assertNever(transcriptsConfigurationGet.provider);
+      }
```

Comment:
Rather than blacklisting, which requires update when we add new transcript, can you move to a whitelist approach with a regular if block?